ambry-network/src/main/java/com/github/ambry/network/http2/Http2MultiplexedChannelPool.java

```diff
@@ -92,13 +94,16 @@
    * @param eventLoopGroup The event loop group.
    * @param http2ClientConfig http2 client configs.
    * @param http2ClientMetrics http2 client metrics.
+   * @param streamChannelInitializer {@link ChannelInitializer} to initilize a stream channel pipeline
    */
   public Http2MultiplexedChannelPool(InetSocketAddress inetSocketAddress, SSLFactory sslFactory,
-      EventLoopGroup eventLoopGroup, Http2ClientConfig http2ClientConfig, Http2ClientMetrics http2ClientMetrics) {
+      EventLoopGroup eventLoopGroup, Http2ClientConfig http2ClientConfig, Http2ClientMetrics http2ClientMetrics,
+      ChannelInitializer streamChannelInitializer) {
     this(new SimpleChannelPool(createBootStrap(eventLoopGroup, http2ClientConfig, inetSocketAddress),
             new Http2ChannelPoolHandler(sslFactory, inetSocketAddress.getHostName(), inetSocketAddress.getPort(),
                 http2ClientConfig)), eventLoopGroup, ConcurrentHashMap.newKeySet(), inetSocketAddress,
-        new Http2ClientConfig(new VerifiableProperties(new Properties())), http2ClientMetrics);
+        new Http2ClientConfig(new VerifiableProperties(new Properties())), http2ClientMetrics,
```

Comment:
why is the http2ClientConfig passed in from the constructor not used here?

Reply:
good catch. http2ClientConfig should be used. I guess it was because refactor.